pytest.ini

```diff
@@ -54,5 +54,7 @@ python_files =
     # Ref: https://docs.pytest.org/en/latest/reference.html#confval-python_files
     # Needed to discover legacy nose test modules:
     Test*.py
-testpaths = test/
+    # Needed to discover embedded Rule tests
+    *Rule.py
+testpaths = test/ lib/ansiblelint/rules/
```

Comment:
This is why I used pyargs in #1009:
https://github.com/ansible/ansible-lint/pull/1028/checks?check_run_id=1062724401#step:11:582

Reply:
I think that flaky failure (happens only on macos, ~1/4) is unrelated to how we pass the test args. Are you sure it does not happen with pyargs?

I asked about pyargs vs testpaths but nobody was able to provide a comparison, yet.